ci/docker/base.ml.Dockerfile

```
@@ -1,6 +1,11 @@
 ARG DOCKER_IMAGE_BASE_TEST
 FROM $DOCKER_IMAGE_BASE_TEST
 
+ENV RAY_INSTALL_JAVA=1
```

```suggestion
ENV RAY_INSTALL_JAVA=1
```

Do we actually need the Ray Java language bindings here?

This will always build the Ray jars for every build job, this is probably not what we want